devfile/devworkspace-operator
# Improve test e2e scenario

### What does this PR do?
Implement test scenario in next way: 
- reuse deployment mechanism we have in Makefile to avoid duplication make deploy;
- create web-terminal workspace from samples;
- waits until workspace is RUNNING;
- check if the current user is able to exec into workpsace-related pod;
- check that another user is not able to exec into workspace-related pod, even if they have pod/exec permissions;
### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/17419
